internal/api/server.go

```diff
@@ -132,6 +132,7 @@ func (s *Server) StartServer() error {
 	pluginGroup := e.Group("/plugin", s.userAuthMiddleware)
 	pluginGroup.POST("/policy", s.CreatePluginPolicy)
 	pluginGroup.PUT("/policy", s.UpdatePluginPolicyById)
+	pluginGroup.POST("/sign", s.SignPluginMessages)
```

Comment:
There is a method call `SignMessages` currently we have , which is mostly inherit from vultiserver
As far as I understand , we will not need it 
You have two choice here, either keep your new `SignPluginMessages` , or update the `SignMessage` to behave as SignPluginMessage , we don't need both

Reply:
Yes, already planning to remove it this week, tracked in https://github.com/vultisig/verifier/issues/89